src/persist-client/src/internal/gc.rs

```diff
@@ -227,6 +228,13 @@ where
 
         let delete_semaphore = Semaphore::new(machine.cfg.gc_blob_delete_concurrency_limit);
 
+        let mut step_start = Instant::now();
```

Comment:
oh nice! this could be a neat little helper in metrics.rs if we wanted. something like:

```rust
pub struct StepTimer{
    step_start: Instant,
}

impl StepTimer {
    pub fn report(&mut self, coutner: &Counter) { ... }
}
```

Reply:
Totally! Something you'd like to see here, or the next time this pattern comes up?

Reply:
up to you!

Reply:
Normally I'm a "factor it out the second time it comes up" type person. It's possible that we could use this style for the existing compaction timings... I may take a stab at that as a followup, if it looks promising.